src/PaymentGateways/Gateways/Stripe/LegacyStripePaymentIntent.php

```diff
@@ -0,0 +1,53 @@
+<?php
+
+namespace Give\PaymentGateways\Gateways\Stripe;
+
+use Give_Stripe_Payment_Intent;
+
+/**
+ * @unreleased
+ */
+class LegacyStripePaymentIntent
```

Comment:
Another candidate for the `LegacyPaymentGateways` domain.

Reply:
Refactored to be a payment intent value object.